x-pack/plugins/siem/public/mock/mock_ecs.ts

```diff
@@ -19,15 +19,15 @@ export const mockEcsData: Ecs[] = [
       action: ['Action'],
       category: ['Access'],
       module: ['nginx'],
-      severity: ['3'],
+      severity: [3],
     },
     source: {
       ip: ['192.168.0.1'],
-      port: ['80'],
+      port: [80],
     },
     destination: {
       ip: ['192.168.0.3'],
-      port: ['6343'],
+      port: [6343],
     },
     user: {
       id: ['1'],
```

Comment:
Should these `user.id`'s be a `number`? Looks like they are currently in `feature-secops`.